package com.example.prova.Support.Eccezioni;

public class PrenotatoException extends Exception{
    private final String msg = "Prenotazione gi√† esistente";

    public PrenotatoException(){
        super();
        System.out.println(msg);
    }
}
